src/main/java/frc/trigon/robot/Subsystems/Roller/Roller.java

```diff
@@ -0,0 +1,91 @@
+package frc.trigon.robot.Subsystems.Roller;
+
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.*;
+
+public class Roller extends SubsystemBase {
```

Comment:
All the commands here should require the roller subsystem. This creates another problem in all of the parallel commands because a parallel command can't have two commands that require the same subsystem. So add a method that removes all the requirements for a command. To do this, you need to return the same command as a FunctionalCommand that does in every step what the given command does. (This should look something like new FunctionalCommand(command::init, etc...))

Reply:
I don't understand what you mean by this.

Reply:
What exactly don't you understand?

Reply:
Any of it. I don't understand what doesn't work and what I need to put in to make it work.

Reply:
Currently, everything will work, but it won't require the roller subsystem. This will create a lot of problems; a command needs to require its subsystem. If you add the requirements to all commands (just adding this at the end of the params), the code will not work since a parallel command can't have 2 commands that require the same subsystem. So need to create a method that removes requirements from a command. I explained how to do that in the first comment.

Reply:
I'm still not sure I fully understand. If you're ever free in the lab, I can come and talk with you in person.

Reply:
Sure, we can meet there on Thursday if you're free